master/internal/api_project.go

```diff
@@ -174,6 +174,12 @@ var defaultRunsTableColumns = []*projectv1.ProjectColumn{
 		Location:    projectv1.LocationType_LOCATION_TYPE_RUN,
 		Type:        projectv1.ColumnType_COLUMN_TYPE_TEXT,
 	},
+	{
+		Column:      "archived",
+		DisplayName: "Archived",
+		Location:    projectv1.LocationType_LOCATION_TYPE_RUN,
+		Type:        projectv1.ColumnType_COLUMN_TYPE_UNSPECIFIED, // no boolean type
```

Comment:
q: would it be possible to add a boolean type in this pr, or is that out of scope?

Reply:
Let's handle it separately because `parentArchived` in runs + metadata is also affected by that change.